tests/flink/flink-security.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load ../utils
+load ../security
+
+@test "SDL T1191 test" {
+    # start container first
+    docker run --name flink-server --detach clearlinux/flink jobmanager
```

Comment:
I think we don't need add tests for flink.
Because all the java based containers will be deleted soon.